packages/fiori/test/pages/ViewSettingsDialog.html

```diff
@@ -26,18 +26,41 @@ <h3> ViewSettingsDialog</h3>
 	<br>
 	<br>
 	<ui5-view-settings-dialog id="vsd">
-		<ui5-li slot="sortItems">Name</ui5-li>
-		<ui5-li slot="sortItems">Position</ui5-li>
-		<ui5-li slot="sortItems">Company</ui5-li>
-		<ui5-li slot="sortItems">Department</ui5-li>
+
+		<ui5-sort-item slot="sortItems" text="Name"></ui5-sort-item>
+		<ui5-sort-item slot="sortItems" text="Position"></ui5-sort-item>
+		<ui5-sort-item slot="sortItems" text="Company"></ui5-sort-item>
+		<ui5-sort-item slot="sortItems" text="Department"></ui5-sort-item>
+
+		<ui5-filter-item slot="filterItems" text="Filter 1">
+			<ui5-filter-item-option slot="values" text="Some filter 1"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 2"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 3"></ui5-filter-item-option>
+		</ui5-filter-item>
+		<ui5-filter-item slot="filterItems" text="Filter 2">
+			<ui5-filter-item-option slot="values" text="Some filter 4"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 5"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 6"></ui5-filter-item-option>
+		</ui5-filter-item>
+		<ui5-filter-item slot="filterItems" text="Filter 3">
+			<ui5-filter-item-option slot="values" text="Some filter 7"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 8"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 9"></ui5-filter-item-option>
+		</ui5-filter-item>
+		<ui5-filter-item slot="filterItems" text="Filter 4">
+			<ui5-filter-item-option slot="values" text="Some filter 10"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 11"></ui5-filter-item-option>
+			<ui5-filter-item-option slot="values" text="Some filter 12"></ui5-filter-item-option>
+		</ui5-filter-item>
 	</ui5-view-settings-dialog>
 
 	<script>
 		btnOpenDialog.addEventListener("click", function () {
 			vsd.show();
 		});
 		vsd.addEventListener("confirm", function(evt) {
-			alert("OK button clicked, returned info is: " + JSON.stringify(evt.detail));
+			// alert("OK button clicked, returned info is: " + JSON.stringify(evt.detail));
```

Comment:
Which of these two should we use? We can remove the other one

Reply:
Done